src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1

```diff
@@ -698,14 +698,10 @@ function Test-ProvidersList
         $job | Wait-Job
         $list1 = $job | Receive-Job
         $list2 = Get-AzureRmNetworkWatcherReachabilityProvidersList -NetworkWatcher $nw -Location "West US" -Country "United States" -State "washington"
-        $list3 = Get-AzureRmNetworkWatcherReachabilityProvidersList -NetworkWatcher $nw -Location "West US" -Country "United States" -State "washington" -City "seattle"
```

Comment:
So, are we no longer testing the `-City` param?

Reply:
Yes. Because there is no more data on city level. Data has been changed

Reply:
Can you mark the City param as deprecated, since users can no longer use it? Also, make note to remove this param in our next breaking change release.

Reply:
it's not deprecated. It's only means that I can be sure that there will be data on city level for specified parameters. There will be data on city level for different countries and regions.

Reply:
@irrogozh Can you make a test that guarantees there will be city data? The `City` parameter  should be tested.

Reply:
No, I can't. That's the problem with data. Another team is working on it....

Reply:
We just query data from Kusto. What's in this data is out of our control...